pkg/forward/setup.go

```diff
@@ -99,32 +99,21 @@ func (f *Forwarder) setupAttackerCapture(bind netip.Addr) error {
 	}
 	f.iface = iface
 
-	handle, err := pcap.NewInactiveHandle(iface.Name)
+	nf, err := nflog.Open(&nflog.Config{
+		Group:    833,
+		Copymode: nflog.CopyPacket,
+		// Push to userspace after 10 ms
+		Timeout: 1,
+	})
 	if err != nil {
-		return fmt.Errorf("create inactive handle: %w", err)
-	}
-	if err := handle.SetImmediateMode(true); err != nil {
-		return fmt.Errorf("set immediate mode: %w", err)
-	}
-	if err := handle.SetTimeout(pcap.BlockForever); err != nil {
-		return fmt.Errorf("set timeout: %w", err)
-	}
-	if err := handle.SetSnapLen(1600); err != nil {
-		return fmt.Errorf("set snap length: %w", err)
+		return err
 	}
 
-	attackerCapture, err := handle.Activate()
-	if err != nil {
-		return fmt.Errorf("activate handle: %w", err)
-	}
-	if err := attackerCapture.SetBPFFilter(fmt.Sprintf("host %s", bind)); err != nil {
-		return fmt.Errorf("set BPF filter: %w", err)
-	}
-	if err := attackerCapture.SetDirection(pcap.DirectionIn); err != nil {
-		return fmt.Errorf("set direction: %w", err)
+	if err := nf.SetOption(syscall.NETLINK_NO_ENOBUFS, true); err != nil {
```

Comment:
I know that this kind of stuff can all be read in the man pages, but comments would generally be great so that someone just trying to understand what this does and why does not have to spend the time.